source/adapters/cuda/adapter.cpp

```diff
@@ -37,7 +37,7 @@ class ur_legacy_sink : public logger::Sink {
   ~ur_legacy_sink() = default;
 };
 ur_adapter_handle_t_::ur_adapter_handle_t_()
-    : logger(logger::get_logger("cuda")) {
+    : logger(logger::get_logger("cuda", /*default_log_level*/ "level:info")) {
```

Comment:
I think `level:error` gives the same amount of info as the old logger used to by default

Reply:
Yes I think you're right. I had `level:error` but then got confused by the wording in internal ticket. Fixed.